src/services/MixService.php

```diff
@@ -64,10 +71,14 @@ public function init()
         $this->rootPath = rtrim(CRAFT_BASE_PATH, '/');
         $this->publicPath = trim($settings->publicPath, '/');
         $this->assetPath = trim($settings->assetPath, '/');
-        $this->manifestPath = implode('/', array_filter([
+        $this->assetFullPath = implode('/', array_filter([
             $this->rootPath,
             $this->publicPath,
             $this->assetPath,
+        ]));
+
+        $this->manifestPath = implode('/', array_filter([
```

Comment:
`array_filter` not needed here since the two path parts are already fully normalized.